src/components/wallet/WalletNetworkGuard.tsx

```
@@ -0,0 +1,97 @@
+  // Don't show if wallet is available on this network
+  if (isAvailable) {
+    return null;
+  }
+
+  // Don't show if no active wallet
+  if (!activeWallet) {
+    return null;
+  }
+
+  // Handle add network action
+  const handleAddNetwork = useCallback(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)